team/guide/issue-triage.md

```diff
@@ -8,65 +8,41 @@ editor: markdown
-- Have a standard set of labels,
-- Have the appropriate fields set in our GitHub project.
-- Be in a milestone.
+- Everyone creating issues should do their best to triage them at creation time.
+- [Team](/team) members also [rotate](/team/guide/comms-assignee) responsibility for catching issues that were not triaged during creation.
 
-New issues are automatically created with the `status: triage` label and are automatically added to our GitHub project.
+## How to triage one issue
 
-## Who should triage issues?
-Any member of [the Mathesar core or community team](/team) is encouraged to triage issues. If you're making an issue, please ensure that it is triaged as well.
+1. **Set required labels**
 
-## How should I triage issues?
-Each step in the triage process is detailed below.
+    | Prefix | Required count | Notes |
+    | - | - | - |
+    | `work:` | 1+ | All `work: design` issues should also be marked `restricted: design team` because only the design team can currently work on those. |
+    | `type:` | 1 | |
+    | `status:` | 1 | _Should not be `status: triage`_ |
 
-### Applying labels
-All issues should have the following labels:
-- One or more `work:` labels. These describe the kind of work involved in the issue (frontend, backend, documentation, design, etc.)
-- One `type:` label. This describes the kind of problem that the issue is solving (bug, enhancement, etc.) This may be automatically present based on the issue template.
-  - There's also a `question` label for issues that are questions and not problems.
-  - `type: meta` issues collect other issues and are not meant to be worked on directly.
-- One `status:` label describing the current status of the issue. If the issue has been triaged, this should not be `status: triage`.
+1. **Set optional labels**
+  
+    Scan through the full labels list and apply other labels as necessary. Learn more about the [meaning of all our labels](https://github.com/centerofci/mathesar/labels).
 
-There are also other labels you should consider applying:
-- `priority` labels for issues that are either urgent or are for future consideration.
-- `restricted:` labels for issues that are not open to the community.
-  - All `work: design` issues should also be marked `restricted: design team` because only the design team can currently work on those.
-  - Consider marking complicated issues or issues that touch the fundamental architecture of Mathesar as `restricted: maintainers`.
-- `good first issue` and `help wanted` labels for issues that are particularly suitable for community contribution.
-  - Ensure that `help wanted` is applied to all issues of this type, but only apply `good first issue` in addition to `help wanted` if the issue is approachable for new and inexperienced contributors.
-- `affects:` labels for issues that touch technical debt, architecture, DX, or UX.
-- Any other labels that apply from our [repository's configured labels](https://github.com/centerofci/mathesar/labels).
```

Comment:
This is all documentation that explains the meaning of the labels. I'd rather put that in the GitHub label descriptions so that it's more visible

Reply:
Makes sense. I do think it makes sense to have some documentation on when it makes sense to mark an issue as `help wanted` or `good first issue` and to make sure to mark things as `help wanted` even if the `good first issue` label is applied.

Criteria to mark things as help wanted:
- The issue should be sufficiently detailed so that someone outside the team can feasibly contribute to it.
- The issue should not be an urgent priority or blocking issues that are next on our roadmap (contributors sometimes take a few weeks to complete tasks)

Issues marked good first issue should be suitable for people new to the codebase in addition to the above criteria.

Reply:
I see.

>  `good first issue` ... should not be an urgent priority or blocking issues 

That's a good point and was not previously clear to me. I still find the `help wanted` label a bit confusing though. The fact that we have `help wanted` and `restricted` seem somewhat redundant. I don't think it's worth spending a ton of time to hammer out though because these things will always be blurry. But it does seem somewhat important to me that our issues are able to be clear enough to document within GitHub alone -- not requiring any external wiki to explain, because that'll be harder for people to discover. I think we can probably fit an abridged version of that text you wrote into the label description. It'll be a bit long, but I think that's okay. Would you like me to write up some proposed label descriptions to set in GitHub directly? I can run them by you before setting them.

Reply:
`help wanted` is a default issue label used by GitHub, people use it across repositories to find issues they can contribute to. I think it's important to keep it. Same with `good first issue`.

I think there's a middle ground between `help wanted` and `restricted`. `restricted` means there's some important reason why an outside contributor cannot work on the issue. `help wanted` means we're actively seeking help and don't plan to work on it ourselves soon. Issues with neither label could be worked on by contributors if they wanted, they're just not as friendly.

Feel free to update the label descriptions, you don't have to run them by me before setting them. I can take a look once you've set them.

The audience of the issue triage page is maintainers and community team members, I was trying to put everything you'd need to triage an issue in one place. The audience of the label descriptions is everybody, including new contributors, I'm not sure if both audiences need the same information. I don't have a specific suggestion there, just mentioning it in case it's helpful as you update the labels and/or this PR.

Reply:
> help wanted is a default issue label used by GitHub, people use it across repositories to find issues they can contribute to. I think it's important to keep it. Same with good first issue.

Ahh ok. I did't realize that.

Reply:
Might be good to document that too.